base/uk.ac.stfc.isis.ibex.opis/resources/Common Scripts/sendOne.py

```diff
@@ -0,0 +1,4 @@
+from org.csstudio.opibuilder.scriptUtil import PVUtil
+
+pv = widget.getPV(); 
+pv.setValue(1)
```

Comment:
Is there really no way to avoid a separate script for each different value we want to send to a PV?

Reply:
0 and 1 are common values to want to send. I can't think of a better way immediately without having to rely on another PV. It's done like this in most OPIs, I just extracted it into a separate script.